src/controller/tourCMSFactoryController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace TourCMS\OnBoarding\Controller;
+
+use PhpParser\Error;
+use TourCMS\OnBoarding\Model\tourCMSFactory;
+use TourCMS\OnBoarding\Controller\mustacheController;
+use TourCMS\OnBoarding\Helper\redisWrapper;
+
+class tourCMSFactoryController
```

Comment:
I guess here it is a problem.
A factory should return objects.
A controller should manage the flow.

Does not make sense a tourCMS FactoryController which seems to work like a service or a library to work with TourCMS API.
Feel free of arrange a meeting and we can discuss